torch/backends/_nnapi/serializer.py

```diff
@@ -418,6 +418,8 @@ def add_tensor_operand_for_weight(self, tensor):
             buf_num,
             offset,
             tsize))
+        if dim_order == DimOrder.CHANNELS_LAST:
```

Comment:
Can you just leave a comment here?  Something like

Because the NNAPI operand will have shape NHWC, we need to permute the torch tensor to the same dimension order to ensure that the data will be properly laid out after we call contiguous on it.